.github/workflows/ci.yml

```diff
@@ -86,7 +86,7 @@ jobs:
       - uses: dtolnay/rust-toolchain@master
         with:
           toolchain: ${{ matrix.rust }}
-      - run: cargo test --features test-with-podman,with-file-async-tokio
+      - run: cargo test --features with-file-async-tokio
```

Comment:
How come we don't have that feature there anymore?

Reply:
Well, I was thinking that it didn't make sense any longer if the tests were not going to be directly in the source tree.  But that was fairly early on so I'm pretty neutral again, I can add it back.

Reply:
I don't understand. The idea is to have a controlled environment to verify the built packages are recognized as valid rpms by the existing tooling. So I really would like these to remain. It's a form of acceptance test.

Reply:
The tests do remain, they're just not flagged on the feature.  I was thinking there was a better way to control whether the tests were run or not but after searching it appeared there wasn't, particularly.

Reply:
And what speaks against keeping the feature flags as a unique discriminator that have a host dependency?

Reply:
It just feels weird to have a feature that applies only to tests

Reply:
It's a solution to a problem, if you have a more elegant solution, perfect! Otherwise please keep the feature flag